examples/advanced_bot.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: UTF-8 -*-
+
+from fbchat import Client, Group, Message
+
+
+class Bot(Client):
+
+    def on_message(self, message):
+        thread = message.thread
+
+        # Prevent the bot from answering the bot's own messages
+        if message in self.sent_messages:
```

Comment:
This could be a nice feature to have in general, eg. as `listen(dont_answer_own=True)`, or something like that.
This entire `on_message` could then have been split into several `on_text` and `on_file`

Reply:
Maybe an `__init__` parameter? `enable/disable` method? Or maybe even a function decorator?
Actually, thinking about it, I think I like the idea of a function decorator for this quite a lot.